synapse/lib/remcycle.py

```diff
@@ -926,7 +939,22 @@ def response_nommer(resp):
 
         # Construct the request object
         req = nyx.buildHttpRequest(api_args)
-
-        self.web_client.fetch(req, callback=response_nommer)
+        self._web_queue.put((req, response_nommer))
```

Comment:
you should probably cut out the queue entirely (for perf) by requesting web_client.fetch as the add_callback:
```
web_loop.add_callback( self.web_client.fetch, req, callback=response_nommer )
```